Slim/Formats/Movie.pm

```diff
@@ -325,11 +325,10 @@ sub setADTSProcess {
 			$offset += 3 if $data == 0x80 || $data == 0x81 || $data == 0xfe;
 			$offset++;
 			$data = unpack("N", substr($$bufref, $pos + $offset, 4));
-			$codec->{freq_index} = ($data >> 23) & 0x0f;
+			$codec->{freq_index} = ($data >> 23) & 0x0f;						
+			$codec->{channel_config} = ($data >> 19) & 0x0f;						
 			$codec->{object_type} = $data >> 27;
-			# Fix because Touch and Radio cannot handle ADTS header of AAC Main.
-			$codec->{object_type} = 2 if $codec->{object_type} == 5; 
-			$codec->{channel_config} = ($data >> 19) & 0x0f;
+			$codec->{object_type} = ($data >> 10) & 0x1f if $codec->{object_type} == 5 || $codec->{object_type} == 29;
```

Comment:
Does this change belong in this PR? Or is it something that could be applied independently?

Reply:
no, candidly, it's totally independent but I want lazy because I discovered that issue while working on that PR

Reply:
Would it be wort being back-ported to 8.1?

Reply:
It can, but it's not mandatory. It works like that, it's just not as explicit / clean

Reply:
Let's leave it in 8.2 only